run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+SRC="/app/extra/wt.tar.gz"
+
+# Unload launcher into data directory
+tar -xv --gzip -f $SRC -C $XDG_DATA_HOME
+
+# Export icon
+mkdir "/app/extra/export/share/icons/128x128/apps/"
+mv "$XDG_DATA_HOME/WarThunder/launcher.ico" "/app/extra/export/share/icons/128x128/apps/net.gaijin.WarThunder.png"
```

Comment:
all of this belong to the apply_extra that is supposed to be run after the download, which happen once.

Reply:
This must occur, as the game launcher uses the directory it is placed in as data storage.

/app/bin is write only